CQUI.modinfo

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Mod id="1d44b5e7-753e-405b-af24-5ee634ec8a01" version="1.0">
+<Mod id="1d44b5e7-753e-405b-af24-5ee634ec8a01" version="2.0">
```

Comment:
there has been version 4.0 before... I am afraid we may run into version conflict at some point in the future (once we get to 4.0).  maybe bump to 5.0 then?
I would expect Steam Workshop to reject uploads that would overwrite existing versions... although not sure about that after discovering that you can upload a mod with the same ID without extra permissions :)